src/problem_bank_scripts/problem_bank_scripts.py

```diff
@@ -1230,3 +1230,10 @@ def backticks_to_code_tags(html):
     html = html.replace("\\`", "`")  # Replace escaped backticks
     return html
 
+def validate_header(header_dict):
+    url = "https://raw.githubusercontent.com/open-resources/learning_outcomes/main/Masterlist.csv"
+    learning_outcomes = pd.read_csv(url)
+    # check if header is sanctioned
+    topic_list = learning_outcomes["Topic"].unique().tolist()
+    if header_dict["topic"] not in topic_list:
```

Comment:
```suggestion
    if header_dict["topic"] not in topic_list and header_dict["topic"] != "Template":
```
It makes more sense to me to just special case the Template type than add it as a learning outcome topic at least for now, since there's only one

Reply:
Right now there is only one so it should be fine, but in the case that there are multiple special cases for question topics it would be better to have them listed in the topics list.